apps/ejabberd/src/mod_push_service_mongoosepush.erl

```diff
@@ -123,6 +123,44 @@ http_notification(Host, Method, URL, ReqHeaders, Payload) ->
 %% Helper functions
 %%--------------------------------------------------------------------
 
+%% Create notification for API v1
+make_notification(v1, _Notification, #{<<"silent">> := <<"true">>}) ->
```

Comment:
Do you think we should remove it only from MIM or also from MongoosePush? To be honest I've made backward compatibility also in order to showcase that it's possible, how to do it and to verify that everything needed is in place and ready to use.